pkgs/development/python-modules/python3-xlib/default.nix

```diff
@@ -0,0 +1,13 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+buildPythonPackage rec {
+  pname = "python3-xlib";
+  version = "0.15";
+  doCheck = false;
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-3EJF865KpZScHREu5HI5Aa3jepZyG6lkXyv6VuWzg/g=";
+  };
+}
```

Comment:
We already have this package on nixpkgs, please use it instead: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/xlib/default.nix

Reply:
What a travel, huh

I had to patch two packages to get this working xD